packages/modules/bookmark/src/bookmark-provider.ts

```diff
@@ -0,0 +1,154 @@
+import { ModuleType } from '@equinor/fusion-framework-module';
+
+import {
+    EventModule,
+    FrameworkEvent,
+    FrameworkEventInit,
+} from '@equinor/fusion-framework-module-event';
+
+import { BookmarkClient } from './client/bookmarkClient';
+
+import { BookmarkModuleConfig } from './configurator';
+import { Observable, Subscription } from 'rxjs';
+import { Bookmark, CreateBookmark } from './types';
+
+export interface IBookmarkProvider {
+    readonly config: BookmarkModuleConfig;
+    readonly bookmarkClient: BookmarkClient;
```

Comment:
hide from interface, cast when generating config

Reply:
we talked about me testing private, it did not work, protected seems to work but should i go for public and like you say remove for interface? @odinr